.ci.yaml

```diff
@@ -8,9 +8,40 @@
 enabled_branches:
   - master
 
+platform_properties:
+  windows:
+    properties:
+      caches: >-
+        [
+          {"name": "vsbuild", "path": "vsbuild"},
+          {"name": "pub_cache", "path": ".pub-cache"}
+        ]
+      dependencies: >
+        [
+          {"dependency": "certs"}
+        ]
+      device_type: none
+      os: Windows
+
 targets:
   - name: Windows Plugins
-    builder: Windows Plugins
-    postsubmit: false
```

Comment:
For my own understanding: what is controlling presubmit vs postsubmit in these configs? Or is "both" the default?

Reply:
True is the default, so both run by default. The config links to https://cs.opensource.google/flutter/cocoon/+/master:CI_YAML.md which is the config documentation.